### DISCUSSION TITLE
Alchemy Plus

### DISCUSSION THREAD

**Discussion by syclonix** [alandtse/skyrim_vr_address_library#27] [Ideas]
[Alchemy Plus](https://www.nexusmods.com/skyrimspecialedition/mods/80882)
Improves the alchemy system with dynamic models, better item names, subtracts value for impurities, and rounds magnitudes to decrease inventory clutter.

[Source Code](https://github.com/Exit-9B/AlchemyPlus)
Permissions: This work is licensed under the [MIT License](https://github.com/Exit-9B/AlchemyPlus/blob/main/LICENSE)
CommonLib: I'm not sure how to check whether it uses CommonLib, sorry.

**Comment by alandtse** [comment]
Parapets does do their own VR ports. Have you asked?

**Comment by syclonix** [comment]
Thanks Alan, I will ask!

**Comment by syclonix** [comment]
Alan confirmed he will eventually release a VR port. 

On a separate note, I just want to say thank you again Alan for all the awesome work you do for the VR players. Wishing you a very merry christmas and happy holidays friend.
